package.json

```diff
@@ -13,7 +13,7 @@
     "cypress-open": "cd proxy && yarn cypress-open",
     "cypress-run": "cd proxy && yarn cypress-run",
     "copy-build": "mkdir -p build && yarn copy-root && yarn copy-legacy-assets && yarn copy-ui-css",
-    "copy-legacy-assets": "cp -Ru legacy/dist/assets build/",
+    "copy-legacy-assets": "cp -R legacy/dist/assets build/",
```

Comment:
Driveby to allow the build to work on macos (`-u` is not a valid `cp` flag on macos). I think this should be fine to remove, but do let me know if it was needed for a particular reason.